qutebrowser/config/configdata.yml

```diff
@@ -766,7 +766,6 @@ content.webrtc_ip_handling_policy:
       - disable-non-proxied-udp: WebRTC should only use TCP to contact peers or
             servers unless the proxy server supports UDP. This doesn't expose
             any local addresses either.
-  default: all-interfaces
```

Comment:
This shouldn't have been removed.

Reply:
It was marked as a duplicate key in my editor. But yeah, it's not related to the changes.